pandas/core/generic.py

```diff
@@ -4283,8 +4283,16 @@ def _align_series(self, other, join='outer', axis=None, level=None,
 
         Parameters
         ----------
-        cond : boolean %(klass)s or array
-        other : scalar or %(klass)s
+        cond : boolean %(klass)s, array or callable
+            If cond is callable, it is computed on the %(klass)s and
+            should return boolean %(klass)s or array.
+            The callable must not change input %(klass)s
```

Comment:
Added tags, and currently rendered like below.

![2016-04-30 1 55 14](https://cloud.githubusercontent.com/assets/1696302/14923234/01df7b68-0e77-11e6-9280-fff7388a5139.png)